cvat/apps/quality_control/quality_reports.py

```diff
@@ -2648,6 +2656,22 @@ def _check_project_quality(cls, *, project_id: int) -> int:
         return ProjectQualityCalculator().compute_report(project=project_id).id
 
 
+class QualitySettingsManager:
+    def get_project_settings(self, project: Project) -> models.QualitySettings:
+        return project.quality_settings
```

Comment:
Is this function really needed? Seems like it just adds an extra layer of indirection.